contour/gradle-mvn-push.gradle

```diff
@@ -45,44 +82,26 @@ afterEvaluate { project ->
             mavenDeployer {
                 beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-                pom.groupId = GROUP
-                pom.artifactId = POM_ARTIFACT_ID
-                pom.version = VERSION_NAME
-
                 repository(url: getReleaseRepositoryUrl()) {
                     authentication(userName: getRepositoryUsername(), password: getRepositoryPassword())
                 }
                 snapshotRepository(url: getSnapshotRepositoryUrl()) {
                     authentication(userName: getRepositoryUsername(), password: getRepositoryPassword())
                 }
 
-                pom.project {
-                    name POM_NAME
-                    packaging POM_PACKAGING
-                    description POM_DESCRIPTION
-                    url POM_URL
-
-                    scm {
-                        url POM_SCM_URL
-                        connection POM_SCM_CONNECTION
-                        developerConnection POM_SCM_DEV_CONNECTION
-                    }
-
-                    licenses {
-                        license {
-                            name POM_LICENCE_NAME
-                            url POM_LICENCE_URL
-                            distribution POM_LICENCE_DIST
-                        }
-                    }
-
-                    developers {
-                        developer {
-                            id POM_DEVELOPER_ID
-                            name POM_DEVELOPER_NAME
-                        }
-                    }
-                }
+                configurePom(pom)
+            }
+        }
+    }
+
+    tasks.create("installLocally", Upload) {
+        configuration = configurations.archives
+
+        repositories {
+            mavenDeployer {
+                repository(url: "file://${rootProject.buildDir}/localMaven")
+
+                configurePom(pom)
             }
         }
     }
```

Comment:
this task isn't needed, can be deleted

Reply:
Wouldn't we ever want to install locally? or can we do it without this task?
We've used that a few times with OkHttp, that was handy.

Reply:
this is used by paparazzi and sqldelight to install their plugins to fake local maven repo for integration tests.  i don't think it's needed here.

Reply:
Cool